testsuite/common/src/main/java/io/strimzi/testsuite/oauth/common/TestUtil.java

```diff
@@ -93,4 +93,32 @@ public static void logStart(String msg) {
         System.out.println("========    "  + msg);
         System.out.println();
     }
+
+    public static int findFirstMatchingInLog(List<String> log, String regex) {
+        int lineNum = 0;
+        Pattern pattern = Pattern.compile(prepareRegex(regex));
+        for (String line: log) {
+            if (pattern.matcher(line).matches()) {
```

Comment:
It seems a bit weird to have `prepareRegex()` and then use `Matcher.matches()`, rather than just using `Matcher.find()`. Using `find()` would be the preferred way to do this, but if there's a good reason for it then I think it at least deserves a comment.

Reply:
You're absolutely right. That's a dumb code. I'll remove the redundant prepareRegex() and use matches()